context/src/main/java/io/grpc/Context.java

```diff
@@ -719,8 +719,9 @@ public void detach(Context toAttach) {
       uncancellableSurrogate.detach(toAttach);
     }
 
+    // Visible for testing
     @Override
-    public boolean isCurrent() {
+    boolean isCurrent() {
```

Comment:
It's not public on the parent class. Being public is a bug, and we should fix it.